sdks/python/apache_beam/runners/interactive/sql/utils_test.py

```diff
@@ -76,17 +76,23 @@ def test_replace_single_pcoll_token(self):
         replaced_sql, 'SELECT * FROM PCOLLECTION WHERE a=1 AND b=2')
 
   def test_pformat_namedtuple(self):
-    self.assertEqual(
-        'ANamedTuple(a: int, b: str)', pformat_namedtuple(ANamedTuple))
+    expected = pformat_namedtuple(ANamedTuple)
+    self.assertIn('ANamedTuple(', expected)
+    self.assertIn('a: int', expected)
+    self.assertIn('b: str', expected)
 
-  @unittest.skip("BEAM-14288")
   def test_pformat_namedtuple_with_unnamed_fields(self):
-    self.assertEqual(
-        f'OptionalUnionType(unnamed: {repr(Optional[Union[int, str]])})',
-        pformat_namedtuple(OptionalUnionType))
+    expected = pformat_namedtuple(OptionalUnionType)
+    self.assertIn('OptionalUnionType(unnamed: ', expected)
+    self.assertIn('Union', expected)
+    self.assertIn('int', expected)
+    self.assertIn('str', expected)
+    self.assertIn('NoneType', expected)
```

Comment:
Using assertIn seems a little brittle (e.g. `Union[int,my_custom_string_type,float64,NoneType]` would pass as well), but I'm probably being nitpicky here.

Reply:
I think it's fine for these functions since they just pretty print stuff. I could also enumerate all possible combinations of the parameters and then assertIn those.

Sending a commit to reverse test changes for dict and use enumerated values for asserting non-deterministic formatting.